### PR TITLE
Updates welcome message to ask new user to update slack profile

### PR DESCRIPTION
**Pull request by DefCon-007**

### CODE REVIEW

**Comment by athityakumar** [comment]
The changes look good to me. Merging this in. But seriously, `pikachu` branch? :wink:

File: scripts/welcome.coffee [hunk 1]
```diff
@@ -36,6 +36,7 @@ welcome_message_3 = [
   " Why don't you tell us a bit about you?"
 ]
 
+welcome_message_4 = " Also it will be better if you update your slack profile with your real name,contact number etc. "
```
Comment (hargup) [inline scripts/welcome.coffee]: "it will be **helpful**" sounds better.

**Comment by DefCon-007** [comment]
@athityakumar : I was feeling lazy to think of a "valid" name for the branch 😛

**Comment by icyflame** [comment]
@athityakumar @hargup 's comment was ignored and there are serious formatting and grammatical errors in this merge. Please review more rigorously, or wait for more reviews before merging.

**Pull request merged**